### PR TITLE
Story/vspc 42

### PR DESCRIPTION
**Pull request by renilJoseph**
please verify the code.

### CODE REVIEW

**Comment by jdamerow** [comment]
Please see my previous comments. Also, please reopen previous pull requests to keep the comment history.

**Comment by renilJoseph** [comment]
Previous branch was deleted, so cannot reopen it.
As per comment, added font awesome globe icon, changed c:url to href, set external link to open in new window and set the color to red. Please verify.

**Comment by jdamerow** [comment]
Jenkins, test this please.

File: vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp [hunk 1]
```diff
@@ -42,6 +42,29 @@ function drawLinks() {
...
+    
+    <c:forEach items="${externalLinkList}" var="link">
+    {
+        var posX = parseInt($("#space").css('margin-left')) + $("#space").position().left; 
+        var posY = $("#space").position().top;
+        var link = $('<a></a>');
+        link.attr('href', "${link.externalLink.externalLink}");
+        link.attr('target', "_blank");
+        
+        var linkDisplay = $('<span data-feather="navigation-2" class="fa fa-globe"></span>');
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp]: did you notice that the icon is not shown? you need to remove the feather icon, or the globe icon won't show up.

Comment (renilJoseph) [reply vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp]: yes. I missed that. Thank you.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Pull request merged**